.travis.yml

```diff
@@ -96,9 +96,11 @@ script:
   - make -k -j2
   - |
     if [[ "${BUILD}" =~ ^(coverage)$ ]]; then
-      : ; else
+      : # no-op
+    else
       if [[ "${BUILD}" =~ ^(snippet)$ ]]; then
-        ctest . --output-on-failure]; else
+        ctest . --output-on-failure
+      else
         ctest . -j2 --output-on-failure
       fi
```

Comment:
here you kept the semantic?

Reply:
Yes, travis is just a bash script, so there are no steps I can constraint by the build type (if that's what you meant)